roles/octavia/tasks/generate_resources.yml

```diff
@@ -152,3 +152,43 @@
     cloud: atmosphere
     image: "{{ octavia_amphora_image_name }}"
   register: _octavia_amphora_image
+
+- name: Deploy Amphora jumpbox
+  run_once: true
+  block:
```

Comment:
Since this will become 2 tasks only, maybe just make two tasks here and that's it (with `run_once: true`)